framework/include/mfem/problem/MFEMProblem.h

```diff
@@ -71,9 +69,9 @@ class MFEMProblem : public ExternalProblem
   void setMesh();
 
   /**
-   * Initialise the required ProblemOperator used in the Executioner to solve the problem.
+   * Add a ProblemOperator to the executioner used to solve the problem.
```

Comment:
this no longer does what the doxygen says it does

Reply:
Now removed... my hope is that we end up adding this method back in to support custom `ProblemOperators` (that may be useful to specify predefined physics, for example), but realistically this will require a change in the associated `Action` and new `MooseObjects` to support this - outside the scope of this PR.